product_pack/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_pack_line_employee,product.pack.line (se module README on why sale user is used here),model_product_pack_line,base.group_user,1,1,1,1
```

Comment:
This mustn't use sale_user. Check permissions for products, and put them accordingly. And add new one on `sale_product_pack` if needed.